pkg/innerring/innerring.go

```diff
@@ -1181,7 +1181,7 @@ func (s *Server) restartFSChain() error {
 		return fmt.Errorf("FS chain config reinitialization: %w", err)
 	}
 
-	if err = s.resetEpochTimer(true); err != nil {
+	if err = s.resetEpochTimer(false); err != nil {
```

Comment:
Seems like this flag is always `false` now.

Reply:
this flag is `afterNewEpochEvent`, so it is obviously `true` for resetting after new epoch event: https://github.com/nspcc-dev/neofs-node/blob/903c9d95018c8687820f4ab23f6cfdccdc74fd45/pkg/innerring/state.go#L201